### PR TITLE
Adapt storage to applicants

### PR DESCRIPTION
**Pull request by Hou-Rui**
Now you all can view and test your work with the storage for applicants.

### CODE REVIEW

File: src/main/java/com/eva/MainApp.java [hunk 1]
```diff
@@ -82,25 +83,32 @@ private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
         ReadOnlyEvaDatabase<Person> initialPersonData;
```
Comment (Ben-Hanan) [inline src/main/java/com/eva/MainApp.java:83]: Not sure if it's ok to delete the person database. If we can do so without the application collapsing we should remove it if it is redundant! (of course this means updating all the tests) KIV

File: src/main/java/com/eva/model/Model.java [hunk 2]
```diff
@@ -150,4 +162,45 @@
...
+
+    /**
+     * Deletes the given person.
+     * The person must exist in the address book.
+     */
+    void deleteApplicant(Applicant target);
+
+    /**
+     * Adds the given person.
+     * {@code person} must not already exist in the address book.
```
Comment (Ben-Hanan) [inline src/main/java/com/eva/model/Model.java:187]: @code applicant, address book -> eva storage

Comment (Ben-Hanan) [reply src/main/java/com/eva/model/Model.java:187]: the word "address book" appears multiple times in javadocs at multiple places

File: src/main/java/com/eva/model/Model.java [hunk 3]
```diff
@@ -150,4 +162,45 @@
...
+     */
+    void addApplicant(Applicant person);
+
+    /**
+     * Replaces the given person {@code target} with {@code editedPerson}.
+     * {@code target} must exist in the address book.
+     * The person identity of {@code editedPerson} must not be the same as another existing person in the address book.
```
Comment (Ben-Hanan) [inline src/main/java/com/eva/model/Model.java:194]: not sure if we wanna change it to editedApplicant, editedStaff. Can choose to change or not up to u

File: src/main/java/com/eva/model/person/applicant/InterviewDate.java [hunk 4]
```diff
@@ -61,6 +54,6 @@ public boolean isCompleted() {
 
     @Override
     public String toString() {
-        return interviewDate.toString();
+        return DateUtil.dateToString(interviewDate);
```
Comment (Ben-Hanan) [inline src/main/java/com/eva/model/person/applicant/InterviewDate.java:57]: May wanna change the way interview dates are represented. (I think this is for Nikhila?)

**Review by Ben-Hanan — COMMENTED** [review]
Overall good quality. Just a few kinks in the javadocs.

**Comment by Nikhilalalalala** [comment]
LGTM! The changes Ben advised on interviewdate, naming from person to staff and addressbook to eva, I will implement them in my next pr.

**Pull request merged**